client/lib/abtest/active-tests.js

```diff
@@ -73,6 +73,14 @@ module.exports = {
 		defaultVariation: 'drake',
 		allowExistingUsers: true,
 	},
+	statsTabsLikesNudge: {
+		datestamp: '20160409',
+		variations: {
+			dataInformedInsideChart: 10,
+			noNudge: 90
+		},
+		defaultVariation: 'noNudge'
+	},
```

Comment:
This test as implemented will only be shown to new users who sign up after the `datestamp` date. Is that your intent? That's fine if it is, and more "pure" as it won't include people who are like "oh look, something new, I wonder what happens if I click it", but it also means we're testing a _much, much_ smaller set of people, so we'd need a lot more time to determine significance.